packages/sx.js/src/strategies/evm/apeGas.ts

```diff
@@ -0,0 +1,220 @@
+import { defaultAbiCoder, Interface } from '@ethersproject/abi';
+import { Contract } from '@ethersproject/contracts';
+import { Provider } from '@ethersproject/providers';
+import ApeGasVotingStrategy from './abis/ApeGasVotingStrategy.json';
+import DelegateRegistryAbi from './abis/DelegateRegistry.json';
+import Multicall3Abi from './abis/Multicall3.json';
+import SpaceAbi from '../../clients/evm/ethereum-tx/abis/Space.json';
+import {
+  ClientConfig,
+  Propose,
+  Strategy,
+  StrategyConfig,
+  Vote
+} from '../../clients/evm/types';
+import { VotingPowerDetailsError } from '../../utils/errors';
+
+const API_URL = 'https://apevote.api.herodotus.cloud';
+const MULTICALL3_ADDRESS = '0xcA11bde05977b3631167028862bE2a173976CA11';
+
+async function getCurrentDelegatedVotingPower({
+  provider,
+  delegateRegistry,
+  delegationId,
+  voterAddress
+}: {
+  provider: Provider;
+  delegateRegistry: string;
+  delegationId: string;
+  voterAddress: string;
+}) {
+  const delegateRegistryInterface = new Interface(DelegateRegistryAbi);
+  const multicall = new Contract(MULTICALL3_ADDRESS, Multicall3Abi, provider);
+
+  const calls = [
+    {
+      target: delegateRegistry,
+      allowFailure: false,
+      callData: delegateRegistryInterface.encodeFunctionData('delegation', [
+        voterAddress,
+        delegationId
+      ])
+    },
+    {
+      target: delegateRegistry,
+      allowFailure: false,
+      callData: delegateRegistryInterface.encodeFunctionData('getDelegators', [
+        voterAddress,
+        delegationId
+      ])
+    }
+  ];
+
+  const [delegationResult, getDelegatorsResult]: {
+    returnData: string;
+  }[] = await multicall.aggregate3(calls);
+
+  if (!delegationResult || !getDelegatorsResult) {
+    throw new Error('Missing data from DelegateRegistry');
+  }
+
+  const delegators: string[] = [];
+  delegators.push(
+    ...delegateRegistryInterface.decodeFunctionResult(
+      'getDelegators',
```

Comment:
`getDelegators` function doesn't have any gas limits? can return any number of delegators? or it is always one delegator?

Reply:
It's a view, I'm not sure if there are gas/computation limits for views.

This should return all delegators, it uses uint256 so it shouldn't overflow: https://github.com/snapshot-labs/delegate-registry/pull/2/files

Maybe @pscott is aware if there are some limitations we should know about.

Reply:
There is gas limit for view too, it's not specific to function it's a global settings, the more computation you do and more gas it consume. In Snapshot.js we have a class Multicaller which break down multicall in multiple requests. I dont think here it's needed since we only compute data for a single address at once.